experimenter/evaluate_pipeline_new.py

```diff
@@ -0,0 +1,199 @@
+import itertools as it
+import json
+import yaml
+import os
+import parser
+
+from typing import Any, List, Tuple
+from uuid import UUID
+from experimenter import config, utils
+
+from d3m import cli as d3m_cli
+from d3m.contrib.pipelines import K_FOLD_TABULAR_SPLIT_PIPELINE_PATH as k_fold_split_path
+from d3m.contrib.pipelines import FIXED_SPLIT_TABULAR_SPLIT_PIPELINE_PATH as fixed_split_path
+from experimenter.databases.d3m_mtl import D3MMtLDB
+
+
+def save_pipeline_run_to_d3m_db(pipeline_run_path: str):
+    """ 
+    Saves a pipeline run document to the d3m database.
+
+    Parameters
+    ----------
+    pipeline_run_path : path_like str
+        path to pipeline_run document
+
+    Returns:
+    ----------
+    TODO
+
+    Raises:
+    ----------
+    TODO
+    """
+    d3m_db = D3MMtLDB()
+    with open(pipeline_run_path) as pipeline_data:
+        pipeline_run = yaml.full_load(pipeline_data)
+    return D3MMtLDB().save_pipeline_run(pipeline_run)
+
+def evaluate_pipeline_on_problem(pipeline: str=None,
```

Comment:
What do you think of renaming this file to be just `runtime.py` and renaming this function to just `evaluate`?

Reply:
I think that will be good.